scripts/ci/init.sh

```diff
@@ -1,42 +1,87 @@
 #!/bin/sh
 
+set -e
+set -vx
+
+set_env() {
+    AGENT_ID=$(hostname | sed -e 's/.*[^0-9]//')
+    export AGENT_ID=${AGENT_ID:-100}
+
+    WORKSPACE_DIR=$(dirname "$0")/../..
+    export WORKSPACE_DIR=$(cd "$WORKSPACE_DIR" && pwd -P)
+
+    export X1_PREFIX=$(basename "$0" .sh)
+    export X1_VAGRANT_DIR="$WORKSPACE_DIR"/"$X1_PREFIX"
```

Comment:
Nit: this looks simpler:
```suggestion
    export X1_VAGRANT_DIR="$WORKSPACE_DIR/$X1_PREFIX"
```

Reply:
Fixed